examples/E2EBert/README.md

```diff
@@ -0,0 +1,116 @@
+# An End-2-End Deep Learning Workflow with BERT PreTrained Model
+
+An `end-2-end` workflow describing how model training is done,followed by storing all the relevant information leading to model deployment and
+testing. A pretrained BERT model is used to illustrate the workflow.
+
+## Finetuning the BERT Pretrained Model
+The code, adapted from this [repository](https://github.com/maknotavailable/pytorch-pretrained-BERT/blob/master/pytorch_pretrained_bert/modeling.py), 
+is almost entirely dedicated to model training, with the addition of a single mlflow.pytorch.autolog() call to enable automatic logging of params, metrics, and models,
+including the extra files, followed by saving the finetuned model along with extra artifact files such as the vocabulary file and class mapping file, which are essential to make the model 
+work and help in transforming the model outputs into corresponding labels respectively.
+
+## Package Requirement
+
+Install the required packages using the following command
+
+`pip install -r requirements.txt`
+ 
+ 
+ ### Running the code
+To run the example via MLflow, navigate to the `mlflow-torchserve/examples/BertNewsClassification_E2E` directory and run the command
```

Comment:
@karthik-77 Please fix the folder name, it does not match the actual name of the folder

Reply:
have changed the folder name in readme file